readme.md

```diff
@@ -72,6 +98,59 @@ Temporal is a [durable execution system](https://temporal.io/blog/building-relia
 
 - [iWF](https://github.com/indeedeng/iwf) - DSL workflow framework.
 
+## Videos
+
+### Why Temporal
+
+- [Replay Keynote](https://www.youtube.com/watch?v=W0Ygep6iCJY) - Durable execution, Nexus, and Cloud.
+- [Glovo: Fault tolerant distributed microservices with Temporal](https://youtu.be/6lSuDRRFgyY)
+- [TheDevConf: Fault Tolerant, Distributed Microservices Orchestration with Temporal](https://www.youtube.com/watch?v=6T6zVZHU7_Q) - Introduction to Temporal ([slides](https://temporal-intro-and-demo.netlify.app/)).
+- [State of Affairs or Affairs of State](https://www.youtube.com/watch?v=2P_aXee2qh4) - Stateless vs stateful architecture and why Temporal.
+- [Designing a Workflow Engine from First Principles](https://temporal.io/blog/workflow-engine-principles/) - The internal system design of Temporal Server.
+
+### Using Temporal
+
+- [A Guided Tour of Temporal’s New Web UI](https://www.youtube.com/watch?v=iCT4mml8mdM)
+- [Time Travel Tests! Testing Async Workflows, Signals, and Queries with Temporal](https://www.youtube.com/watch?v=-GKxFDQSlEU)
+- [Temporal Service and Application Architecture](https://youtu.be/r5bnp5bQHI4)
+- [Sidekiq, Outbox, SAGAs and Best Practices for Distributed Transactions!](https://www.youtube.com/watch?v=Rr4DnHmy6eE)
+- [How to get workflow failure info using Temporal SDKs](https://www.youtube.com/watch?v=0QuWgqwmr-s)
+- [Keeping Workflow Developers Afloat](https://www.youtube.com/watch?v=yeoawVIn060)
+- [Temporal @ Datadog](https://www.youtube.com/watch?v=LxgkAoTSI8Q)
+- [Workflow versioning](https://www.youtube.com/watch?v=kkP899WxgzY)
+- [Polyglot Microservices Orchestration](https://www.youtube.com/watch?v=LSXP_o6sTic)
+
+## Podcasts
+
+- [Flagsmith](https://flagsmith.com/podcast/temporal-w-co-founder-and-ceo-maxim-fateev-and-head-of-product-ryland-goldstein/) - Maxim & Ryland explain about Temporal and answer questions (50 min, Jun 19, 2022).
+- [Stack Overflow Podcast: Run your microservices in no-fail mode](https://twitter.com/StackOverflow/status/1536740460789739520) - Maxim & Dominik explain Temporal (22 min, Jun 14, 2022).
```

Comment:
Presumably this bulleted list of podcasts (and the other resources) are ordered? (from your top recommendation down?) If not, consider putting the shorter podcast first (this one being 22 min as opposed to 50)

Reply:
yes, mixture of top rec and most relevant